e2e-playwright/process-services-cloud/specs/people-cloud.e2e.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import { test, expect } from '../fixtures/page-initialization';
+
+test.describe.configure({ mode: 'parallel' });
+
+test.describe('People component stories tests', () => {
+    test('Valid Preselected Users', async ({ processServicesCloud, peopleComponent }) => {
+        const expectedUsersName = `
+            first-name-1 last-name-1  cancel
```

Comment:
Why do we have cancel?

Reply:
This 'cancel' is the `x` next to the user naming (which remove user from list in the storie)